src/modules/SharedModule/components/AuthContext/AuthContext.tsx

```diff
@@ -0,0 +1,33 @@
+
+import { jwtDecode } from 'jwt-decode'
+import { createContext, useEffect, useState } from 'react'
+
+     export const AuthContext = createContext(null);
+
+     export default function AuthContextProvider (props:any) {
+
+         const [loginData, setLoginData] = useState('');
+
+         let baseUrl = 'https://upskilling-egypt.com:3003/api/v1'
```

Comment:
What is the difference between having the baseURL in a context and in a constants file?
In other words, can't we put the baseURL in a separate file and import it in any component?